AzureCalling/Controllers/CallViewController.swift

```diff
@@ -515,8 +515,8 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
             }
 
             participantView.updateDisplayName(displayName: participant.displayName)
-            participantView.updateMuteIndicator(isMuted: participant.isMuted)
             participantView.updateActiveSpeaker(isSpeaking: participant.isSpeaking)
```

Comment:
I think we can move the logic here to call `updateActiveSpeaker(false)` when `participant.isMuted == true`. That way we keep `ParticipantView.updateMuteIndicator` simpler and only concerned with updating the UI according to the argument.

Reply:
Updated to no longer have the logic in the participant view